src/platform/Linux/bluez/BluezEndpoint.cpp

```diff
@@ -208,10 +203,13 @@ static gboolean BluezCharacteristicAcquireNotifyError(BluezGattCharacteristic1 *
     return TRUE;
 }
 
+// TODO: When using with BlueZ >= 5.80, this method can be removed. Also, the GetBluezConnectionViaDevice
```

Comment:
Is this an actual TODO (i.e. are we planning to remove) or a note?

I am thinking if actual TODO we should add a github issue with details. We could also maybe IFDEF on the bluez version if that is available.

Reply:
To be honest I'm not sure... However, when using with bluez >= 5.80 this will be basically a dead code...
So maybe indeed it's not a TODO but more like a NOTE

Reply:
Changed to NOTE